src/Stratis.Bitcoin.Features.PoA/PoANetwork.cs

```diff
@@ -180,10 +180,57 @@ public PoANetwork()
                 throw new Exception("No keys for initial federation are configured!");
             }
 
+            this.RegisterRules(this.Consensus);
             this.RegisterMempoolRules(this.Consensus);
         }
 
-        private void RegisterMempoolRules(IConsensus consensus)
+        protected virtual void RegisterRules(IConsensus consensus)
```

Comment:
Just confirming this are exactly he same rules as before right?

Reply:
Yes, and  @zeptin also confirmed.